### DISCUSSION TITLE
Add support for MGRS coordinates

### DISCUSSION THREAD

**Discussion by telluroid** [organicmaps/organicmaps#4235] [Ideas]
MRGS coordinates are not only used by the military, but also by multiple search and rescue organizations. It would be great if it would be possible to enter coordinates in that format into the search dialog and to also copy/share coordinates in that format.

**Comment by biodranik** [comment]
Which apps are supporting it at the moment?

**Reply by telluroid** [reply]
Mapy.cz does, but searching for coordinates only works in online mode. Copying coordinates in that format works without internet connection.

**Reply by telluroid** [reply]
Osmand does support it as well.

**Comment by telluroid** [comment]
UTM coordinates could maybe be implemented at the same time, since MGRS is based on UTM.

**Reply by biodranik** [reply]
See also https://github.com/organicmaps/organicmaps/issues/4715
